portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
+	test AssertTotalAmountOfViewsInAllCards {
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Add a new user in DXP") {
+			JSONUser.addUser(
+				userEmailAddress = "ac@liferay.com",
+				userFirstName = "ac",
+				userLastName = "ac",
+				userScreenName = "ac");
+		}
+
+		task ("Sync Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
+		}
+
+		task ("Interation with page") {
+			ACUtils.navigateToSitePageWithUser(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
```

Comment:
Hey @calvinctvo, I think doing more views might be better to check a number that is different from 1 to validate more hard the bug. What do you think?